thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/grouping/AlertGrouperFactory.java

```diff
@@ -41,6 +41,8 @@ private static AlertGrouper fromStringType(String type) {
       return DUMMY_ALERT_GROUPER;
     case DIMENSIONAL:
```

Comment:
What DIMENSIONAL type do? Its difficult to interpret the difference just by name.